.circleci/config.yml

```diff
@@ -315,7 +315,10 @@ jobs:
 
   docs:
     docker:
-      - image: buildpack-deps:artful
+      - image: buildpack-deps:bionic
+    environment:
+      DEBIAN_FRONTEND: noninteractive
```

Comment:
Can this be moved up to ``defaults``? Not sure whether there can be an ``environment`` field in ``defaults``, but in general it makes sense to have this defined in all runs.